lib/sparql.jison

```diff
@@ -810,7 +812,7 @@ GroupConcatSeparator
 Literal
     : String -> createTypedLiteral($1)
     | String LANGTAG  -> createLangLiteral($1, lowercase($2.substr(1)))
-    | String '^^' iri -> createTypedLiteral($1, $3)
+    | String '^^' iri -> createTypedLiteral($1, $3.value)
```

Comment:
This shouldn't be necessary. I'd change the implementation of `createTypedLiteral` directly to accept a term as second argument.

Reply:
That is a cleaner solution, thanks for the suggestion.